cluster/external/provider.sh

```diff
@@ -7,3 +7,7 @@ function _kubectl(){
 function seed_images(){
   echo "seed_images is a noop for external provider"
 }
+
+function up() {
+  echo "using external cluster"
+}
```

Comment:
You've got an artifact here that's causing you issues

Reply:
I am not sure I follow, make cluster-up for an external cluster is basically a noop, but we are echoing that we are using an external cluster to let the user know. What am I missing?

Reply:
```function verify()```  https://github.com/kubevirt/containerized-data-importer/pull/831/files#diff-11a543d943c2b3d40534dccbf45d4449R11

cluster up call "verify()" at L#11, but the external provider doesn't have a verify function; as a result cluster-up in that configuration fails.  See my notes in the up.sh file, easy to fix.